src/serlio/modifiers/PRTModifierAction.cpp

```diff
@@ -1190,8 +1034,8 @@ MStatus PRTModifierAction::createNodeAttributes(const RuleAttributeSet& ruleAttr
 			case prt::Attributable::PT_BOOL: {
 				const bool value = mGenerateAttrs->getBool(fqName.c_str());
 				if (attrTrait.first == AttributeTrait::ENUM) {
-					mEnums.emplace_front();
-					MCHECK(addEnumParameter(attrTrait.second.mAnnot, node, attr, p, value, mEnums.front()));
+					mEnums.try_emplace(p.mayaFullName);

```

Comment:
optimization: you could use the return value of `try_emplace` to avoid the lookup on the next line

Reply:
using return value now.